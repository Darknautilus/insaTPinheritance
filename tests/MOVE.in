R rectangle1 56 46 108 4536 
L l172v 56 46 108 4536 
C circle1 12 15 45
OA agg1 rectangle1 circle1
Oa agg2 l172v agg1
MOVE rectangle1 -45 70
LIST
MOVE rectangle2 40 78
LIST
MOVE agg2 -10 1
LIST
