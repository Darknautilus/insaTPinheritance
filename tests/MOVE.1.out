OK
OK
OK
OK
OK
OK
OK
OK
C circle 2 4 3
L line 3 4 4 5
PL poly 4 5 5 6 6 7 7 8
R rectangle 5 7 7 9
